packages/Datepicker/src/components/CalendarGrid.tsx

```diff
@@ -1,64 +1,64 @@
-import * as React from 'react';
+import * as React from "react";
 
-import { getWeeksInMonth } from '@internationalized/date';
-import { useCalendarGrid, useLocale } from 'react-aria';
-import { CalendarState, RangeCalendarState } from '@react-stately/calendar';
+import { getWeeksInMonth } from "@internationalized/date";
+import { useCalendarGrid, useLocale } from "react-aria";
+import type { CalendarState, RangeCalendarState } from "@react-stately/calendar";
 
-import CalendarCell from './CalendarCell';
+import CalendarCell from "./CalendarCell";
 
 type State = CalendarState | RangeCalendarState;
 
 interface CalendarGridProps {
-  state: State;
-  highlightToday?: boolean;
+    state: State;
+    highlightToday?: boolean;
```

Comment:
I would have named this highlightCurrentDay

Reply:
What is your reasoning? I know that Datepickers tend to use the word "Today". This was previously named by Franck. I think I will keep as is for now. If you feel strongly about this, we can ask to rename it once he comes back.

Reply:
No strong opinion. Although today could not be today for someone elsewhere versus current day feels more "universal". Anyhow just a thought ;)